src/cxx_stubs/SFWindow_stub.cpp

```diff
@@ -85,7 +85,7 @@ caml_sfWindow_createFromHandle(value ml_handle, value ml_settings)
 
     sf::Window *window;
     window = new sf::Window;
-    window->create(Nativeint_val(ml_handle), settings);
+    window->create((sf::WindowHandle)Nativeint_val(ml_handle), settings);
```

Comment:
I used a C-style cast as we are casting `void*` and calling C code. I have nothing against `static_cast`.